### PR TITLE
#1 Layout for design and front page

### PR DESCRIPTION
**Pull request by triszt4n**

### CODE REVIEW

**Comment by triszt4n** [comment]
Problem: Flowbite told me to install npm package flowbite via npm -> https://flowbite.com/docs/getting-started/rails/

So the docker setup and Dockerfile should change too (?)

I loved this flowbite package, makes tailwind a lot easier (kind of a chakra but for simple SSR sites)

File: Gemfile.lock [hunk 1]
```diff
@@ -219,6 +223,8 @@ GEM
       sprockets (>= 3.0.0)
     stimulus-rails (1.1.0)
       railties (>= 6.0.0)
+    tailwindcss-rails (2.0.14-x64-mingw32)
```
Comment (triszt4n) [inline Gemfile.lock:226]: Yeah, windows shit

**Review by triszt4n — COMMENTED** [review]
Also: I gave up on making every component's class list into one, as there's a shit ton of them, and it's easier to just ctrl+c ctrl+v stuff from flowbite. Sry.

**Review by bbucsy — APPROVED** [review]
Ehh tried avoiding nodejs in true Rails 7 style.
But hey as long as it works :shrug:

Design looks really good tho
:100: 

I pushed a commit to update the docker config

**Pull request merged**